project/build.gradle

```diff
@@ -48,7 +49,9 @@ dependencies {
 	testImplementation "org.testcontainers:junit-jupiter:1.20.4"
 	testImplementation 'org.testcontainers:mysql:1.19.0'
 	testImplementation "org.springframework.kafka:spring-kafka-test:3.3.0"
+	testImplementation 'io.gatling.highcharts:gatling-charts-highcharts:3.13.1'
```

Comment:
Trying to install this dependency causes `Plugin [id: 'io.gatling.gradle', version: '3.13.1'] was not found in any of the following sources:`